src/applications/static-pages/shared/ExpandableOperatingStatus.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './sass/_m-operating-status.scss';
+
+const ExpandableOperatingStatus = props => {
+  const [open, setOpen] = useState(false);
+  const handleOnclick = e => {
+    if (!props.extraInfo) return;
+    e.target.classList.toggle('active');
+    const content = e.target.nextElementSibling;
+    if (content.style.display === 'block') {
+      content.style.display = 'none';
+      setOpen(false);
+    } else {
+      content.style.display = 'block';
+      setOpen(true);
+    }
```

Comment:
Rather than manipulating the DOM directly here, can we compute the class style dynamically based on the `open` state?